easybuild/easyconfigs/b/BLAST+/BLAST+-2.14.0-gompi-2022b.eb

```diff
@@ -36,6 +36,7 @@ dependencies = [
     ('libpng', '1.6.38'),
     ('libjpeg-turbo', '2.1.4'),
     ('LMDB', '0.9.29'),
+    ('cpio', '2.15'),
```

Comment:
I think this is only required during the installation, so it can be moved to the build dependencies. I've made a PR to change that: https://github.com/TopRichard/easybuild-easyconfigs/pull/1. Also applied the same change to one older version of BLAST+.